vtexdocs/devportal
# Feature/algolia search

#### What is the purpose of this pull request?

Implement the search with a link to Algolia indices and create the search/results components for **large screens**.

[Documented Here](https://www.notion.so/vtexhandbook/Implementar-Algolia-no-componente-de-front-e3bbfcd8345f4bba90a44697835a20d2) :book: 

#### What problem is this solving?

The absence of a search feature.

#### How should this be manually tested?
[Figma component](https://www.figma.com/file/Lx6sXdrw9KEvtSEKWttmv8/Developer-Portal?node-id=1791%3A116653) :art: 

Go to [this page](https://deploy-preview-84--elated-hoover-5c29bf.netlify.app/) and check de search component
- The style:
  - Colors, fonts, icons, color changes on hover, etc...
- The behavior: 
  - on hover, the search bar will change its size and, on click, modify its background color and icon/label positions
  - When the user clicks inside of the search bar, the search input will focus on.
  - If the user types something inside of the search input, the results box needs to appear
  - If the user clicks outside of the component (search-input and results-box), the results box will close
  - If the user clicks on a result, the results box will close and he will be redirected.
  - If a query have 7 or more results, the see All button will appear and, on clicking inside, the user will be redirected to the search page

#### Screenshots or example usage

#### Types of changes
